2-mandatory/9-cheap-diner.js

```diff
@@ -19,7 +19,17 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+  mealArray.sort((a, b) => a.price - b.price);
+
+  if (mealArray.length == 0) {
+    return "Nothing :(";
+  } else if (mealArray.length == 1) {
+    return `${mealArray[0].name}`;
+  } else {
+    return `${mealArray[1].name}`;
+  }
+}
```

Comment:
Hey Rabie. I was struggling to solve this exercise. I clearly understand where I went wrong. Thanks!!